scipy/stats/tests/test_stats.py

```diff
@@ -380,6 +383,7 @@ def test_pROUNDROUND(self):
 
 
 @skip_xp_backends(cpu_only=True)
+@skip_xp_backends("jax.numpy", reason="data-apis/array-api-extra#134")
```

Comment:
This is a skip for jax, but the reason is a dask-specific issue (that has been resolved in dask 2025.2.0)?

Reply:
Right, I just happened to noticed the mistake, too.

Reply:
```suggestion
```